tests/test_diffing/test_diff_writer.py

```diff
@@ -9,7 +9,7 @@
 import yaml
 from deepdiff import DeepDiff
 
-from sceptre.diffing.diff_writer import DiffWriter, DeepDiffWriter, deepdiff_json_defaults, DiffLibWriter
+from sceptre.diffing.diff_writer import DiffWriter, DeepDiffWriter, deepdiff_json_defaults, DiffLibWriter, ColouredDiffLibWriter
 from sceptre.diffing.stack_differ import StackDiff, DiffType, StackConfiguration
```

Comment:
You should add a test suite on the Coloured writer here. It doesn't have to test everything you didn't implement. Really, I think it would just need four tests, really one for each possible code path of the code you created.